package.json

```diff
@@ -1,27 +1,35 @@
 {
-  "name": "subscription-portal",
-  "version": "0.1.0",
-  "private": true,
-  "scripts": {
-    "dev": "next dev --turbopack",
-    "build": "next build",
-    "start": "next start",
-    "lint": "next lint"
-  },
-  "dependencies": {
-    "next": "^15.3.3",
-    "react": "^19.0.0",
-    "react-dom": "^19.0.0"
-  },
-  "devDependencies": {
-    "@eslint/eslintrc": "^3",
-    "@tailwindcss/postcss": "^4.1.10",
-    "@types/node": "^24.0.3",
-    "@types/react": "^19.1.8",
-    "@types/react-dom": "^19.1.6",
-    "eslint": "^9",
-    "eslint-config-next": "^15.3.3",
-    "tailwindcss": "^4.1.10",
-    "typescript": "^5.8.3"
-  }
+    "name": "subscription-portal",
+    "version": "0.1.0",
+    "private": true,
+    "scripts": {
+        "dev": "next dev --turbopack",
+        "build": "next build",
+        "start": "next start",
+        "lint": "next lint",
+        "format": "prettier --write './**/*.js' './**/*.mjs' './**/*.ts' './**/*.tsx'",
+        "check-format": "prettier -c './**/*.js' './**/*.mjs' './**/*.ts' './**/*.tsx'",
+        "check": "npm run lint && npm run check-format"
+    },
+    "dependencies": {
+        "next": "^15.3.3",
+        "react": "^19.0.0",
+        "react-dom": "^19.0.0"
+    },
+    "devDependencies": {
+        "@eslint/eslintrc": "^3",
+        "@tailwindcss/postcss": "^4.1.10",
+        "@types/node": "^24.0.3",
+        "@types/react": "^19.1.8",
+        "@types/react-dom": "^19.1.6",
+        "eslint": "^9",
+        "eslint-config-next": "^15.3.3",
+        "eslint-config-prettier": "^10.1.5",
+        "prettier": "3.6.0",
+        "tailwindcss": "^4.1.10",
+        "typescript": "^5.8.3"
+    },
+    "prettier": {
+        "tabWidth": 4
```

Comment:
I went with a `tabWidth` of 4, since that's what we're using in sonora (all the other settings in sonora are prettier defaults at this point).